TravelCompProject/travelcompapp/models.py

```diff
@@ -3,29 +3,48 @@
 
 # Create your models here.
 # Need to add active_user boolean.
+
+
 class UserDetails(models.Model):
     user_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
     guid = models.UUIDField(default=uuid.uuid4)
     first_name = models.CharField(max_length=200, blank=False)
     last_name = models.CharField(max_length=200, blank=False)
     picture_url = models.URLField(max_length=200, blank=True)
```

Comment:
Picture url will be more than 200 characters. We should remove max_length and add unique = True